src/tribler-core/tribler_core/modules/category_filter/family_filter.py

```diff
@@ -18,21 +18,21 @@ def initTerms(filename):
     searchterms = set()
 
     try:
-        with open(filename, 'r') as f:
+        with open(filename) as f:
             lines = f.read().lower().splitlines()
 
             for line in lines:
                 if line.startswith('*'):
                     searchterms.add(line[1:])
                 else:
                     terms.add(line)
-    except IOError:
-        raise IOError(u"Could not open %s, initTerms failed.", filename)
+    except OSError:
```

Comment:
**question:** Is it the correct replacement?

Reply:
`IOError` and `OSError` are the same since Python 3.3:

```
>>> IOError is OSError
True
```